assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/vision/config.py

```diff
@@ -23,6 +23,8 @@ class Tasks(_CustomEnum):
     MM_OBJECT_DETECTION = "image-object-detection"
     MM_INSTANCE_SEGMENTATION = "image-instance-segmentation"
 
+    IMAGE_TEXT_TO_TEXT = "image-text-to-text"
```

Comment:
I don't think, you need this changes. Can you please check where it's used. This folder is only for OD, IS model only, just like llava folder. We need to refactor this to rename vision to detection task.

Reply:
Done.